architecture/consumption-architecture-dependency-management.md

```diff
@@ -0,0 +1,148 @@
+# Introduction
+## Context and Scope
+This document captures technologies that are hosted in OpenSSF and some technologies in CNCF. The technical stack in this document demonstrates how OSS security technologies make software supply chain more secure through dependency management. 
+
+The goals of this document are:
+1. To help open source producers quickly navigate the OSS security technology landscape, discover, adopt and contribute to technical initiatives. 
```

Comment:
"navigate" is an intermediate goal, not really a final goal. It also doesn't make it clear that our focus is on OpenSSF projects vs. all possible technology. How about:

To help open source software producers discover, adopt as appropriate, and contribute to technical initiatives of OpenSSF (and related foundations) to improve security.

Reply:
Thank you @david-a-wheeler for taking the time. Updated the goal per your suggestion.